werkit/aws_lambda/deploy.py

```diff
@@ -147,6 +147,23 @@ def perform_update_code(
     client = boto3.client("lambda", region_name=aws_region)
 
     def update(code_arguments):
+        # this waiter is needed to work around an issue in
+        # JWT_RSA_AWS_CUSTOM_AUTHORIZER lambda specifically, where calling the
```

Comment:
Just thought we should generalize this

```suggestion
        # certain lambdas, where calling the
```